packages/devtools_app/lib/src/analytics/provider.dart

```diff
@@ -2,11 +2,15 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-abstract class AnalyticsProvider {
-  bool get isGtagsEnabled;
-  bool get shouldPrompt;
-  bool get isEnabled;
-  void setUpAnalytics();
-  void setAllowAnalytics();
-  void setDontAllowAnalytics();
+import 'package:flutter/foundation.dart';
+
+class AnalyticsProvider {
+  final ValueListenable<bool> analyticsEnabled = ValueNotifier<bool>(false);
+  final ValueListenable<bool> shouldPrompt = ValueNotifier<bool>(false);
+  void enableAnalytics() {}
+  void disableAnalytics() {}
+  void setUpAnalytics() {}
 }
+
+Future<AnalyticsProvider> get analyticsProvider async => _provider;
```

Comment:
getter seems async for no reason.

Reply:
this is the stub version of this getter, but the real html getter needs to be async. Removed `async` and changed return type to `FutureOr` so that we can still call `await analyticsProvider` in the code.